tests/FI.API.SignTool.Tests/KnownData.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+
+namespace FI.API.SignTool.Tests
+{
+    public class KnownData
+    {
+        // NOTE: These files are the same as the ones provided with this repo in the `Data` folder
+        //       Copied into this repo to ensure they are always available to the test runner
```

Comment:
Not following here, are you talking about projects not repos?

Reply:
Yep, good spot - I'll change